packages/components/MenuButton/jest.config.js

```diff
@@ -1,2 +1,7 @@
 const { configureReactNativeJest } = require('@uifabricshared/build-native');
-module.exports = configureReactNativeJest('android');
+module.exports = configureReactNativeJest('android', {
+  transform: {
+    '^.+\\.(js|ts|tsx)?$': 'babel-jest',
+    '^.+\\.svg$': '<rootDir>/svgTransform.js',
+  },
+});
```

Comment:
Why was this needed?

Reply:
To fix an issue with SVG translate. But it didn't help.  After I imported SVG the tests started failing. I added it here so if someone would help me he\she will see my solution.

Reply:
I fixed it with moduleNameMapper instead of transform

Reply:
But why is this configuring the android jest module? Is that what win32 uses?

Reply:
(I should also say I don't actually know what jest is or does, so these comments are not review blocking comments but "for my understanding" comments)

Reply:
Talked offline. This is a solution from stack overflow, https://stackoverflow.com/questions/58603201/jest-cannot-load-svg-file#

Reply:
I saw that all our components use android configuration. I tried to use win32, it didn't work. I also need to read more about how it works :)